packages/wdio-browserstack-service/src/crash-reporter.ts

```diff
@@ -0,0 +1,121 @@
+import logger from '@wdio/logger'
+import type { Capabilities, Options } from '@wdio/types'
+import got from 'got'
+import { createRequire } from 'node:module'
+
+import { DATA_ENDPOINT } from './constants.js'
+import type { BrowserstackConfig, CredentialsForCrashReportUpload, UserConfigforReporting } from './types.js'
+import { DEFAULT_REQUEST_CONFIG, getObservabilityKey, getObservabilityUser } from './util.js'
+
+const log = logger('@wdio/browserstack-service')
+const require = createRequire(import.meta.url)
+const { version: bstackServiceVersion } = require('../package.json')
```

Comment:
This also happens in `launcher.js`, let's have this done once in one place e.g. a `constants.js` and export it for other packages to consume

Reply:
done. moved to constants https://github.com/webdriverio/webdriverio/pull/10183/commits/fb0fbb92db57a39111250de63ee60149885b50aa